### ISSUE TITLE
Invalid media type & Unknown Video Info Key

### ISSUE THREAD

**Issue by PizzaPino** [groggyegg/plugin.video.dramacool#29]
```
2022-05-13 20:27:01.513 T:2785    ERROR <general>: NEWADDON Unknown Video Info Key "__no_default__"
2022-05-13 20:27:01.513 T:2785  WARNING <general>: Invalid media type "/category/korean-drama"
```

I keep getting this error over and over until Kodi crashes for me.

I'm on the latest OSMC (March2022) & Vero4K.

**Comment by groggyegg** [comment]
Fixed in [23e9183](https://github.com/groggyegg/plugin.video.dramacool/commit/23e9183e25bf47c305ee792348ab00be6ed61f95). Could you verify if this solves the issue.

**Comment by PizzaPino** [comment]
yes it's fixed, thanks!
